src/core/lib/security/credentials/jwt/jwt_verifier.c

```diff
@@ -528,11 +528,16 @@ static EVP_PKEY *pkey_from_jwk(grpc_exec_ctx *exec_ctx, const grpc_json *json,
     gpr_log(GPR_ERROR, "Cannot set RSA key from inputs.");
     goto end;
   }
+  /* RSA_set0_key takes ownership on success. */
+  tmp_n = NULL;
```

Comment:
Maybe you need to move line 510 and 511 to around line 498 so that it is before the first goto?